### DISCUSSION TITLE
Feedback course

### DISCUSSION THREAD

**Discussion by noel** [PJUllrich/run-elixir#4] [General]
I found the "Run in Livebook" button at the top of the first page confusing, what am I running, I am just starting.

Further down on the page there is information about using `asdf` locally which is great. All that worked.
Then I am told that Livebook is the Recommended way to run the code. However, not having ever installed Livebook I was not sure if installing it via a DMG is the right way or if I could have done it via the terminal.

**Comment by noel** [comment]
After installing livebook it is running at http://localhost:55986/
Clicking the button at the top of the next page does not open the page in livebook if that is what is supposed to happen.

**Comment by noel** [comment]
It feels like `|>` comes out of nowhere. It hasn't been introduced but it is being used in lines like this:
`Enum.at(details, 1) |> IO.inspect(label: 1)`
Also, it can be confusing how this line produces
`1: 32`

**Comment by noel** [comment]
In the functions section
```
 # This function will match all values that didn't match the function clauses above.
 # This could be integer values outside the range from 0 to 150 or other data types.
 #
 # This is also how you write a one-line function without the do ... end notation.

 def adult?(age), do: raise "Invalid age: #{age}"
```

Running this returns `false`, but I expected it to raise an error
`RunElixir.Checker.adult?(:test)`

**Comment by PJUllrich** [comment]
Awesome feedback, Noel! I’ll fix these tomorrow but keep them coming! ❤️

**Reply by PJUllrich** [reply]
Thank you @noel, I implemented your feedback and I hope it's clearer now 💪
